tests/launchsecurity/sev.go

```diff
@@ -19,23 +19,33 @@ var _ = Describe("[sig-compute]AMD Secure Encrypted Virtualization (SEV)", func(
 		tests.BeforeTestCleanup()
 	})
 
-	It("should start a SEV VM", func() {
-		const secureBoot = false
-		vmi := libvmi.NewFedora(libvmi.WithUefi(secureBoot), libvmi.WithSEV())
-		vmi = tests.RunVMIAndExpectLaunch(vmi, 240)
+	DescribeTable("When starting a SEV or SEV-ES VM",
+		func(isESEnabled bool) {
+			const secureBoot = false
+			dmesgRet := "SEV"
+			if isESEnabled {
```

Comment:
It would be bettter to skip the test if there is no ES-capable node in the cluster. You can refer to the implementation of the function `checks.SkipTestIfNotSEVCapable()` (which is called here in `BeforeEach`) and implement a similar one for ES:

https://github.com/kubevirt/kubevirt/blob/409494e0c5b34f4a1cb5b251954eee3404654c67/tests/framework/checks/skips.go#L117